### PR TITLE
feature: add --label option for container

### PR DESCRIPTION
**Pull request by skoo87**
Signed-off-by: skoo87 <marckywu@gmail.com>

**1.Describe what this PR did**

**2.Does this pull request fix one issue?** 

**3.Describe how you did it**

**4.Describe how to verify it**

**5.Special notes for reviews**

### CODE REVIEW

File: cli/container.go [hunk 1]
```diff
@@ -19,6 +22,17 @@ func (c *container) config() *types.ContainerCreateConfig {
 	// TODO
 	config.Tty = c.tty
 
+	// set labels
+	config.Labels = make(map[string]string)
+	for _, label := range c.labels {
+		fields := strings.SplitN(label, "=", 2)
+		if len(fields) != 2 {
+			continue
```
Comment (allencloud) [inline cli/container.go]: I suggest that we should throw out the validation error message. If no error, user will by default consider his operation was right, but that is not true. 
So I think in cli, we should should report error as soon as possible.

File: cli/create.go [hunk 2]
```diff
@@ -44,6 +44,7 @@ func (cc *CreateCommand) addFlags() {
 	flagSet.BoolVarP(&cc.tty, "tty", "t", false, "Allocate a tty device")
 	flagSet.StringSliceVarP(&cc.volume, "volume", "v", nil, "Bind mount volumes to container")
 	flagSet.StringVar(&cc.runtime, "runtime", "", "Specify oci runtime")
+	flagSet.StringSliceVarP(&cc.labels, "--label", "l", nil, "Set label for a container")
```
Comment (allencloud) [inline cli/create.go]: s/label/labels

File: cli/run.go [hunk 3]
```diff
@@ -51,6 +51,7 @@ func (rc *RunCommand) addFlags() {
 	flagSet.StringVar(&rc.detachKeys, "detach-keys", "", "Override the key sequence for detaching a container")
 	flagSet.BoolVarP(&rc.attach, "attach", "a", false, "Attach container's STDOUT and STDERR")
 	flagSet.BoolVarP(&rc.stdin, "interactive", "i", false, "Attach container's STDIN")
+	flagSet.StringSliceVarP(&rc.labels, "--label", "l", nil, "Set label for a container")
```
Comment (allencloud) [inline cli/run.go]: s/label/labels

**Comment by allencloud** [comment]
> Merging #488 into master will not change coverage.

@Letty5411 I found this PR added integration test. I think CodeCov only do statistics analysis on unit test, right?  Just double check for this. Thanks

File: cli/create.go (hunk 2 above)
Comment (skoo87) [reply cli/create.go]: the usage is "pouch create -l a=b -l 1=2 xxxxx", so I think we should use "label", not "labels".

File: cli/run.go (hunk 3 above)
Comment (skoo87) [reply cli/run.go]: as same as the above comment.

File: cli/create.go (hunk 2 above)
Comment (allencloud) [reply cli/create.go]: Different aspects. I think both are reasonable. 🦁

**Comment by Letty5411** [comment]
@allencloud Yes, codecov only stats the coverage of unit test.

**Comment by allencloud** [comment]
LGTM

**Pull request merged**